project.json

```diff
@@ -12,20 +12,14 @@
     },
     "code-pushup": {
       "command": "npx dist/packages/cli",
```

Comment:
This should already use the new executor ``, but our CI executes the released version. 
Therefor we have to change this in a followup PR.